documentation: ZENKO-1958 general improvements

jonathan-gramain opened the pull request:
* copyedits to Using.md, DESIGN.md, metrics.md, healthcheck.md

* site-CRR copyedits

Original PR from @wabernatScality that this one replaces: https://github.com/scality/backbeat/pull/281

wabernat committed:
documentation: ZENKO-1958 general improvements

* copyedits to Using.md, DESIGN.md, metrics.md, healthcheck.md

* site-CRR copyedits

bert-e commented:
# Hello jonathan-gramain,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/backbeat/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers





Peer approvals *must* include at least 1 approval from the following list:

* @bennettbuchanan

* @rahulreddy

* @vrancurel

* @jonathan-gramain

jonathan-gramain committed:
documentation: ZENKO-1958 remove docs/site-level-crr.md

The process described in this file is not part of Zenko features, so
remove it.

+ minor copyedits

wabernatScality commented:
@bert-e approve

bert-e commented:
# Not Author



I'm afraid I cannot do that, @wabernatscality:

Only the author of the pull request @jonathan-gramain can use this command.

Please **delete** the corresponding comment so I can move on.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers





Peer approvals *must* include at least 1 approval from the following list:

* @bennettbuchanan

* @rahulreddy

* @vrancurel

* @jonathan-gramain

wabernat commented on DESIGN.md:
```diff
@@ -1,250 +1,213 @@
+## Writing Extensions
+
+A Backbeat extension enables adding more functionality to the core
+backbeat asynchronous processor. Asynchronous Replication, for example,
```
lowercase "replication."

wabernat commented on DESIGN.md:
```diff
@@ -1,250 +1,213 @@
+
+There are two ways to approach this:
+
+1. Use Pub/Sub events in addition to the MetaData log in a separate topic.
```
* not 1.

wabernat commented on DESIGN.md:
```diff
@@ -1,250 +1,213 @@
+   - Number of objects in backlog
+   - Number of objects replicated so far etc.
+
+2. Use a decoupled topic in addition to the queue topic. The
```
* not 2.

JianqinWang commented on DESIGN.md:
```diff
@@ -1,250 +1,213 @@
+disaster-recovery setup, on losing Site A, a customer can point to
+Site B for READONLY actions until Site A is back up. Although the
+customer can write to Site B, these writes are not replicated to
+Site A (among other reasons because Site A may have data that is
```
minor/optional: `among other reasons because` -> `among other reasons,` or `among other reasons because xyz` -> `because xyz, among other reasons`.

jonathan-gramain commented:
@bert-e approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/8.1`



The following branches will **NOT** be impacted:

* `development/7.4`
* `development/7.5`
* `development/8.0`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/8.1`



The following branches have **NOT** changed:

* `development/7.4`
* `development/7.5`
* `development/8.0`



Please check the status of the associated issue ZENKO-1958.

Goodbye jonathan-gramain.
